packages/palette-tokens/src/index.ts

```diff
@@ -84,25 +84,35 @@ const tokens = {
     /** 10 black (grey), borders, divider lines, and grey button only */
     black10: "#E5E5E5",
     /** 5 black (light grey), backgrounds only */
-    black5: "#F8F8F8",
-    /** 100 Blue, highlight */
+    black5: "#F2F2F2",
```

Comment:
@deliciousnachos just to confirm, is `#F2F2F2` the correct value for `black5`? That's what's in Notion, but I wasn't sure if Palette was outdated or Notion was outdated :)

Reply:
@mdole Oh interesting! Didn't mean to make any updates to the grays, let's keep what's already in palette #F8F8F8